clientv3/integration/watch_test.go

```diff
@@ -558,3 +558,33 @@ func TestWatchEventType(t *testing.T) {
 		}
 	}
 }
+
+func TestWatchErrConnClosed(t *testing.T) {
```

Comment:
Also needs `TestWatchAfterClose` where the client is closed before calling clientv3.NewWatcher like in the KV close tests